src/Tizen.Applications.Cion/Tizen.Applications/IPayload.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// An abstract class to represent payload.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public abstract class IPayload
+    {
+        private readonly string LogTag = "Tizen.Cion";
+        internal PayloadSafeHandle _handle;
+
+        /// <summary>
+        /// Gets type of the payload.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public abstract PayloadType GetPayloadType();
```

Comment:
Property가 더 선호됩니다.

```suggestion
        public abstract PayloadType PayloadType
        {
            get;
        }
```

하위 클래스는 아래와 같이 override가능합니다.
``` c#
        public override PayloadType PayloadType
        {
            get
            {
                return PayloadType.FilePayload;
            }
        }
```

Reply:
Property 로 수정하였습니다.